### ISSUE TITLE
Error while deobefusating

### ISSUE THREAD

**Issue by AgainPsychoX** [sd-soleaio/javascript-deobfuscator#5]
Source attached (script from 9anime.to website) [source.zip](https://github.com/sd-soleaio/javascript-deobfuscator/files/7604802/source.zip)

Log:
```

> javascript-deobfuscator@1.0.0 start C:\Users\PsychoX\Desktop\deob
> tsc && node dist/run.js

PS C:\Users\PsychoX\Desktop\deob> npm start

> javascript-deobfuscator@1.0.0 start C:\Users\PsychoX\Desktop\deob
> tsc && node dist/run.js

D:\Users\PsychoX\Desktop\deob\dist\modifications\proxies\proxyRemover.js:97
                if (node == scope.node && scope.parent) {
                                  ^

TypeError: Cannot read property 'node' of undefined
    at Controller.leave (D:\Users\PsychoX\Desktop\deob\dist\modifications\proxies\proxyRemover.js:97:35)
    at Controller.__execute (D:\Users\PsychoX\Desktop\deob\node_modules\estraverse\estraverse.js:397:31)
    at Controller.traverse (D:\Users\PsychoX\Desktop\deob\node_modules\estraverse\estraverse.js:491:28)
    at Object.traverse (D:\Users\PsychoX\Desktop\deob\node_modules\estraverse\estraverse.js:713:27)
    at ProxyRemover.replaceProxyFunctionUsages (D:\Users\PsychoX\Desktop\deob\dist\modifications\proxies\proxyRemover.js:75:27)
    at Controller.enter (D:\Users\PsychoX\Desktop\deob\dist\modifications\proxies\proxyRemover.js:86:44)
    at Controller.__execute (D:\Users\PsychoX\Desktop\deob\node_modules\estraverse\estraverse.js:397:31)
    at Controller.traverse (D:\Users\PsychoX\Desktop\deob\node_modules\estraverse\estraverse.js:501:28)
    at Object.traverse (D:\Users\PsychoX\Desktop\deob\node_modules\estraverse\estraverse.js:713:27)
    at ProxyRemover.replaceProxyFunctionUsages (D:\Users\PsychoX\Desktop\deob\dist\modifications\proxies\proxyRemover.js:75:27)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! javascript-deobfuscator@1.0.0 start: `tsc && node dist/run.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the javascript-deobfuscator@1.0.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

**Comment by sd-soleaio** [comment]
Hi, can you upload a sample script this happens on to [pastebin.com](https://pastebin.com/)

**Comment by Keller18306** [comment]
I have the same problem.
Script: https://www.tiktok.com/acrawler/webmssdk.js
This happens when "Replace Proxy Functions" is on.

**Comment by sd-soleaio** [comment]
Should be fixed in 5f3324484a4d14d993e1d6d7d9bcc402eede5ede
Can you check if this fixed the issue for you (can quickly check at https://deobfuscate.io) @Keller18306 @AgainPsychoX

**Comment by Keller18306** [comment]
Yes it is fixed
